From 8a5dbdb38ff882b84a68cb141f1c5479cae8840b Mon Sep 17 00:00:00 2001
From: rugnathr <rugnath.ram@intel.com>
Date: Fri, 23 Nov 2018 15:55:37 +0530
Subject: [PATCH] Fix issue that Virtual keypad ducks down when trying to click
 on it with mouse.

Inputmethod crashed since mAccessibilityManager is
not initialized when clicking Virtual keypad with
mouse. The solution is that initializing
mAccessibilityManager before using it.

Tracked-on: OAM-71763
Signed-off-by: rugnathr <rugnath.ram@intel.com>
---
 src/com/android/inputmethod/latin/car/KeyboardView.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/inputmethod/latin/car/KeyboardView.java b/src/com/android/inputmethod/latin/car/KeyboardView.java
index b237bae..b2741c6 100644
--- a/src/com/android/inputmethod/latin/car/KeyboardView.java
+++ b/src/com/android/inputmethod/latin/car/KeyboardView.java
@@ -289,6 +289,7 @@ public class KeyboardView extends View implements View.OnClickListener {
     public KeyboardView(Context context, AttributeSet attrs, int defStyle) {
         super(context, attrs, defStyle);
         mContext = context;
+        mAccessibilityManager = (AccessibilityManager) context.getSystemService(Context.ACCESSIBILITY_SERVICE);
 
         TypedArray a =
                 context.obtainStyledAttributes(
-- 
1.9.1

